<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminNonVeg</class>
 <widget class="QDialog" name="adminNonVeg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/backgr.PNG);</string>
  </property>
  <widget class="QLineEdit" name="foodName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>DISH..........</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="foodPrice">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>RS.............</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delFood">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>DISH..............</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DELETE A NON-VEG DISH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD A NON-VEG DISH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>UPDATE NON-VEG MENU</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
